build/purger/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG RAIDEN_VERSION
+FROM raidennetwork/raiden:${RAIDEN_VERSION}
+
+COPY requirements.txt /requirements.txt
+RUN /opt/venv/bin/python3 -m pip install -r /requirements.txt
```

Comment:
Why do you use `/opt/venv/bin/python3` here instead of `python3`?

Reply:
That the location of the venv from the raiden container. That way building the image is faster since the raiden requirements are already there.